_posts/2021-01-08-2020-review.md

```diff
@@ -0,0 +1,254 @@
+  authenticators](/docs/current/develop/certificate-authenticator.html)
+* [Salesforce authenticator](docs/current/security/salesforce.html)
+* The query engine and access control SPIs now support injecting row filters and
+  column masks.
+* New syntax for managing permissions (`GRANT/REVOKE` on schema,
+  `ALTER TABLE/SCHEMA/VIEW ... SET AUTHORIZATION`)
+
+## Supported data source
+
+Trino empowers you to use one platform to access all data sources. Connectors
+enable this and we added numerous new connectors:
+
+* [Iceberg](/docs/current/connector/iceberg.html)
+* [Prometheus](/docs/current/connector/prometheus.html)
+* [Oracle](/docs/current/connector/oracle.html)
+* [Pinot](/docs/current/connector/pinot.html)
+* [Druid](/docs/current/connector/druid.html)
+* [BigQuery](/docs/current/connector/bigquery.html)
+* [MemSQL](/docs/current/connector/memsql.html)
+
+All other connectors received a large host of improvements. Let’s just look at
+two popular connectors:
+
+### Hive connector for HDFS, S3 and cloud object storage system
+
+* Complex Hive views, allows seamless integration with Hive or simplifying
+  migration from Hive
+* [ACID transactional tables]({% post_url 2020-06-01-hive-acid %}) with `INSERT`
+  and `DELETE` support
+* [Built-in storage caching](/docs/current/connector/hive-caching.html) and
+  support for [external caching with
+  Alluxio](/docs/current/connector/hive-alluxio.html)
+* New procedures: `system.drop_stats()`, `register_partition()`,
+  `unregister_partition()`
+* Support for [Azure object storage](/docs/current/connector/hive-azure.html)
+* Support for [S3 encrypted files, flexible S3 security mappings and
+  Intelligent-Tiering S3 storage](/docs/current/connector/hive-s3.html)
+
+### Elasticsearch connector
+
+The [Elasticsearch connector](/docs/current/connector/elasticsearch.html)
+received numerous powerful improvements:
+
+* Password authentication
+* Support for index aliases
+* Support for array types, Nested, and IP type
+* Support for Elasticsearch 7.x
+
+## Runtime improvements
+
+Operating and maintaining a Trino cluster takes a significant amount of
+resources. So any work to improve the runtime needs have a significant positive
+impact:
+
+* [Requirement to use Java
+  11](/docs/current/installation/deployment.html#java-runtime-environment), with
+  better GC performance, overall performance, and improved also container
+  support
+* Support for ARM64-based processors to run Trino
+* Support for minimum number of workers before query starts, useful for
+  implementing autoscaling
+* [Data integrity checks for network transfers]({% post_url
+  2020-06-25-data-integrity-protection %})
+
+## Everything else
+
+There is so much more to capture, and you really would have to read all the
+[release notes](/docs/current/release.html#releases-2020) in detail to know it
+all. To safe you from that, here are a few more noteworthy changes:
+
+* JDBC driver backward compatibility tests
```

Comment:
oh dear, these are good!  (except we just had to break compat... )

Reply:
haha ... you want it removed?

Reply:
No, I think it’s important to highlight.